falwa/f90_modules/compute_qref_and_fawa_first.f90

```diff
@@ -26,7 +26,8 @@ SUBROUTINE compute_qref_and_fawa_first(pv, uu, vort, pt, tn0, imax, JMAX, kmax,
   REAL :: qbar(nd,kmax)
 
   pi = acos(-1.)
-  dp = pi/float(jmax-1)
+  dp = pi/float(jmax-1)  !!!  This is dlat
+  dl = 2*pi/float(imax-1)  !!!! pragallva- correction added on Dec 13, 2023. This is dlon
```

Comment:
@Pragallva Now I realize there's a small mistake here. This should be:
```python
dl = 2*pi/float(imax)
```
since 0 and 2 pi are the same point. If you are doing analysis on your side with this version, please make the correction!

Reply:
Oh right! Thanks @csyhuang for noticing the inconsistency. Usually imax is a large number of the order of 100 or 200, so it shouldn't make a huge difference to the area element value, but yes its important to be consistent.